styleguide/source/_patterns/01-molecules/06-components/20-testimonial/testimonial-left.md

```diff
@@ -0,0 +1,7 @@
+---
+el: ".testimonial-left"
+title: "Testimonial 50/50 Left"
+---
+A Testimonial 50/50 Left pattern consists of an Image, a Quote (long text), a Speaker (text), and a Speaker Title (text). There are two variants, a left and a right variant. In this variant, the image is on the left.
```

Comment:
See comments on testimonial.md.

I might also just describe this pattern in terms of being a variant of the other pattern. "The Testimonial 50/50 Left pattern is a variant of the Testimonial 50/50 pattern..."